frontend/src/components/Contexts/AlertContext.tsx

```diff
@@ -29,13 +30,15 @@ export enum AlertType {
     DismissDock,
     DockSuccess,
     AutoScheduleFail,
+    InfoAlert,
 }
 
 const alertTypeEnumMap: { [key: string]: AlertType } = {
     DockFailure: AlertType.DockFail,
     generalFailure: AlertType.RequestFail,
     DockSuccess: AlertType.DockSuccess,
-    autoSchedule: AlertType.AutoScheduleFail,
+    autoScheduleFail: AlertType.AutoScheduleFail,
+    skipAutoMission: AlertType.InfoAlert,
```

Comment:
alertTypeEnumMap mixes between camelCase and PascalCase. Suggest fixing to stick to one. Doesn't need to be part of this PR though